src/main/java/net/sf/jabref/gui/journals/AbbreviationsFile.java

```diff
@@ -1,39 +1,43 @@
 package net.sf.jabref.gui.journals;
 
-import java.io.File;
 import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.OutputStream;
 import java.io.OutputStreamWriter;
-
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import net.sf.jabref.Globals;
 import net.sf.jabref.logic.journals.Abbreviation;
 import net.sf.jabref.logic.journals.JournalAbbreviationLoader;
 
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
 import javafx.beans.property.SimpleListProperty;
 import javafx.collections.FXCollections;
 
-public class AbbreviationsFile extends File {
+public class AbbreviationsFile {
 
     private final SimpleListProperty<Abbreviation> abbreviations = new SimpleListProperty<>(
             FXCollections.observableArrayList());
+    private final Path path;
 
 
-    public AbbreviationsFile(File file) {
-        super(file.getAbsolutePath());
+    public AbbreviationsFile(String filePath) {
+        path = FileSystems.getDefault().getPath(filePath.replaceAll("\\\\", "/"));
```

Comment:
The idea was indeed to escape the backslashes for windows systems...but thanks for the tip